src/com/company/SQlConnectionSetup.java

```diff
@@ -0,0 +1,19 @@
+package com.company.JDBC;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+class SQlConnectionSetup {
+    Connection connection;
+    PreparedStatement preparedStatement;
+    void establishConnection() throws ClassNotFoundException {
+        final String url = "jdbc:mysql://localhost:3306/employee_database?autoReconnect=true&useSSL=false";
```

Comment:
is their any reason to use autoReconnect, useSSL ?

Reply:
I was getting warning like this

Tue Sep 24 15:56:20 IST 2019 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.

so, i did add it.